src/components/app-drawer/AppDrawer.tsx

```diff
@@ -9,10 +9,13 @@ import useConfirm from '~/hooks/use-confirm'
 import { styles } from '~/components/app-drawer/AppDrawer.styles'
 import { PositionEnum } from '~/types'
 
-interface AppDrawerProps extends DrawerProps {
+type PositionFromEnum = Exclude<PositionEnum, 'start' | 'vertical' | 'end'>
```

Comment:
move it to types or just use Exclude<PositionEnum, 'start' | 'vertical' | 'end'> directrly